python/cppsim_wrapper.cpp

```diff
@@ -602,6 +602,7 @@ PYBIND11_MODULE(qulacs_core, m) {
         .def("add_parametric_multi_Pauli_rotation_gate", &ParametricQuantumCircuit::add_parametric_multi_Pauli_rotation_gate, "Add parametric multi-qubit Pauli rotation gate", py::arg("index_list"), py::arg("pauli_ids"), py::arg("angle"))
 
         .def("backprop",&ParametricQuantumCircuit::backprop,"do backprop",py::arg("obs"))
+        .def("backprop_inpro",&ParametricQuantumCircuit::backprop,"do backprop with innder product",py::arg("state"))
```

Comment:
ここで export すべきは `&ParametricQuantumCircuit::backprop_inpro` だと思います．
```suggestion
        .def("backprop_inpro",&ParametricQuantumCircuit::backprop_inpro,"do backprop with innder product",py::arg("state"))
```

Reply:
ほんとだ、ありがとうございました